src/components/Timeline/EventFilter.js

```diff
@@ -5,10 +5,11 @@ import IconButtonStyled from 'components/elements/IconButtonStyled'
 import FilterChips from 'components/elements/FilterChips'
 import ArrowDown from 'material-ui/svg-icons/navigation/arrow-downward'
 import ArrowUp from 'material-ui/svg-icons/navigation/arrow-upward'
-import AutoComplete from 'material-ui/AutoComplete'
-import * as formActions from 'actions/formActions'
-import * as eventActions from 'actions/eventActions'
-import * as filterActions from 'actions/filterActions'
+
+import AutoCompleteMenu from '../../components/elements/AutoCompleteMenu'
+import * as formActions from '../../actions/formActions'
+import * as eventActions from '../../actions/eventActions'
+import * as filterActions from '../../actions/filterActions'
```

Comment:
This change may be unintentional. Revert to the non relative form 'actions/___________' instead of '../../actions/_____'